README.md

```diff
@@ -1 +1,82 @@
-# Mobile-portfolio-setup
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Table of Contents
+
+  * About the Project
+   - Built With
+
+  *Getting Started
+    - Prerequisites
+    - Setup
+    - Install
+    - Usage
+    - Run test
+    - Deployment
+
+  * Authors
+
+  * Contributing
+
+  * Show your support
+
+  * Acknowledgements
+
+  * License
+
+# PORTFOLIO-SETUP
+
+> My first project at microverse with a simple message "Hello Microverse"
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running on your machine, follow these simple steps:
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
```

Comment:
I strongly recommend you add details on how to clone this project so a newbie will always no how to get a local copy.